enos/utils/enostask.py

```diff
@@ -81,6 +82,15 @@ def decorated(*args, **kwargs):
 
             # Constructs the environment
             kwargs['env'] = make_env(kwargs['--env'])
+
+            # Check if the environment has been loaded for the commands
+            # requiring it
+            if fn.__name__ not in ['up', 'deploy', 'info'] and \
```

Comment:
I maybe miss something but it looks like your previous proposition covered more cases (e.g validation for the install-os were made)
That's why I was proposing to have a dedicated decorator whose job was to check some missing / None values in the env before launching the task. What about : 
```
@enostask("""doc""")
@validation(present=['key1', 'key2'], not_None=['key3'])
```
That would pass only if `key1` and `key2` are present and `key3` not None